src/fsharp/FSharp.Core/seq.fsi

```diff
@@ -615,8 +724,22 @@ module Seq =
     /// <exception cref="T:System.Collections.Generic.KeyNotFoundException">Thrown if no element returns true when
     /// evaluated by the predicate</exception>
     /// <exception cref="T:System.ArgumentNullException">Thrown when the input sequence is null</exception>
-    /// 
-    /// <example-tbd></example-tbd>
+    ///
+    /// <example id="find-index-1">
+    /// <code lang="fsharp">
+    /// [1; 2; 3; 4; 5]
```

Comment:
In general, if you find yourself using multiple lines for the actual example, then pre-bind the input `let inp = [ 1 .. 5 ]`  making the actual call smaller until the it  fits nicely on one line

Reply:
Made all could-be-one-line become so.